docker/batch-init/README.md

```diff
@@ -0,0 +1,124 @@
+
+### Building Batch Initialization Docker image
+
+To build the Docker image, copy the `build.sh.example` file provided to a new file, `build.sh`. 
+This script contains the `--squash` flag, which is optional but recommended. You must enable Docker experimentals on the server if you would like to use this. See [Setup Docker experimentals](#Setup-Docker-experimentals). If your system is not enabled to use squash or you would prefer not to use the `--squash` flag you can remove it from your `build.sh`.
+
+### Execute the build
+
+Once you have updated all the build arguments with your appropriate values, execute the build script with:
+```bash
+$ chmod +x build.sh
+$ ./build.sh
+```
+
+### Running the container
+
+To run the Batch Initializer Docker container, copy the `run.sh.example` script to a new file, `run.sh`. The run script will start the Docker container. Before executing the script, update the passed in Docker environment variables within the `run.sh` script. These environment variables should be configured to meet your needs for your particular Batch Initialization execution. Each variable is described in the table below.
+
+| Env Variable               | Description | 
+| ---------------------------|:--------------| 
+| `S3_SUBMISSION_ARCHIVE_PATH` | The S3 object path to the submission archive |       
+| `S3_VALIDATION_BUCKET`       | The S3 bucket where the validation results should be uploaded to | 
+| `S3_VALIDATION_PREFIX`       | The directory to place the validation results in the `S3_VALIDATION_BUCKET` |    
+| `BATCH_NUM_NODES`        	 | The number of nodes to deploy the AWS Batch job on. *This must match the number of nodes entered when deploying the AWS Batch infrastructure via CloudFormation |  
+| `BATCH_JOB_DEFINITION`       | The AWS Batch job definition to use when executing the validation job |  
+| `BATCH_JOB_QUEUE`     		 | The AWS Batch job queue to use when executing the validation job |  
+| `AWS_SNS_TOPIC_ARN`     	 | The AWS SNS topic to push notifications to during the AWS Batch validation job |  
+
+Execute the run script with:
+
+```bash
+$ chmod +x run.sh
+$ ./run.sh
+```
+
+## Batch Scheduler
+
+Batch Scheduler Docker is an extension of the Batch Initializter which can be used for running Batch Initializations on more than one AIF Validation archive submission that is stored on S3. The only requirement for the Batch Scheduler is that all of the AIF submission archives must live in the same S3 object path. You do not need to build or deploy the Batch Initlaizer to use the Batch Scheduler. All Batch Initlizer logic will be included in the build Batch Scheduler Docker image. 
```

Comment:
Typo: Initializter, Initlaizer, and Initlizer should be Initializer.

Reply:
fixed